### ISSUE TITLE
Deprecated timeout notice

### ISSUE THREAD

**Issue by artem-danilov** [rakibtg/SleekDB#216]
Just installed the sleekdb and have added  few lines of code:

```
require 'vendor/autoload.php';

$databaseDirectory = __DIR__ . "/myDatabase";

$newsStore = new \SleekDB\Store("news", $databaseDirectory);
```

and get an depricated notice on the screen

Deprecated: The "timeout" configuration is deprecated and will be removed with the next major update. Set the "timeout" configuration to false and if needed use the set_timeout_limit() function in your own code. in C:\Users\Dell\Desktop\sleekdb\vendor\rakibtg\sleekdb\src\Store.php on line 747

**Comment by EEKimS** [comment]
Also seeing this.

**Comment by Timu57** [comment]
Hi @EEKimS and @artem-danilov 

The deprecation warning you get was implemented because of the following issue: #205 

We will remove the usage of `set_time_limit` within SleekDB. You should use it in your own code if you want/ need it.

You need to set the `timeout` configuration to `false`, to get rid of that warning.
https://sleekdb.github.io/#/configurations

You can also just ignore the warning.
